EIPS/eip-2537.md

```diff
@@ -25,27 +25,29 @@ The motivation of this precompile is to add a cryptographic primitive that allow
 ### Constants
 
 |Name|Value|Comment|
-|         ---         |---    |         ---        |
-|`FORK_TIMESTAMP`     | *TBD* | Mainnet            |
-|BLS12_G1ADD          | 0x0b  | precompile address |
-|BLS12_G1MUL          | 0x0c  | precompile address |
-|BLS12_G1MULTIEXP     | 0x0d  | precompile address |
-|BLS12_G2ADD          | 0x0e  | precompile address |
-|BLS12_G2MUL          | 0x0f  | precompile address |
-|BLS12_G2MULTIEXP     | 0x10  | precompile address |
-|BLS12_PAIRING        | 0x11  | precompile address |
-|BLS12_MAP_FP_TO_G1   | 0x12  | precompile address |
-|BLS12_MAP_FP2_TO_G2  | 0x13  | precompile address |
-
-If `block.timestamp >= FORK_TIMESTAMP` we introduce *nine* separate precompiles to perform the following operations:
+|         ---          |---    |         ---        |
+|`FORK_TIMESTAMP`      | *TBD* | Mainnet            |
+|BLS12_G1ADD           | 0x0b  | precompile address |
+|BLS12_G1MUL           | 0x0c  | precompile address |
+|BLS12_G1MULTIEXP      | 0x0d  | precompile address |
+|BLS12_G2ADD           | 0x0e  | precompile address |
+|BLS12_G2MUL           | 0x0f  | precompile address |
+|BLS12_G2MULTIEXP      | 0x10  | precompile address |
+|BLS12_PAIRING_CHECK   | 0x11  | precompile address |
+|BLS12_PAIRING_PRODUCT | 0x11  | precompile address |
```

Comment:
this would need to be at a different address

Reply:
Thanks! Fixed with c4f2e32